analyzers/Patrowl/Patrowl_GetReport.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "Patrowl_GetReport",
+    "version": "1.0",
+    "author": "Nicolas Mattiocco",
+    "url": "https://github.com/TheHive-Project/Cortex-Analyzers",
+    "license": "AGPL-V3",
+    "description": "Get the current Patrowl report for a fdqn, a domain or an IP address.",
+    "dataTypeList": ["fqdn", "domain", "ip"],
+    "baseConfig": "Patrowl",
+    "config": {
```

Comment:
These conf properties should be made customizable, and declared as ConfigurationItems. Note that the anlyzer should use an API_KEY instead of username/password couple